src/App/Fossa/ManualDeps.hs

```diff
@@ -16,7 +16,8 @@ module App.Fossa.ManualDeps (
   findFossaDepsFile,
   readFoundDeps,
   getScanCfg,
-) where
+)
+where
```

Comment:
[curious] why?

Reply:
That was auto generated when I ran `make fmt`